test_unstructured_ingest/configs/dest.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+# shellcheck disable=SC2034
+
+core_tests=(
+	'azure.sh'
+	'gcs.sh'
+	's3.sh'
+)
+tests_to_ignore=(
+	'notion.sh'
+	'dropbox.sh'
+	'sharepoint.sh'
+)
+tests_to_omit=()
```

Comment:
Checking [CI logs](https://github.com/Unstructured-IO/unstructured/actions/runs/7182374100/job/19559108932?pr=2217#step:7:1135), whole box test is commented out. Maybe we can omit it here instead

Reply:
Not setting credentials for `sharepoint-embed-cog-index.sh` [skips it](https://github.com/Unstructured-IO/unstructured/actions/runs/7182374100/job/19559108932?pr=2217#step:7:1556) implicitly, rather than explicitly omitting it. We could omit that one here as well